packages/clay-core/src/table/Cell.tsx

```diff
@@ -242,6 +251,14 @@ export const Cell = React.forwardRef<HTMLTableCellElement, Props>(
 							</Layout.ContentCol>
 						)}
 
+						{isLoading && (
+							<Layout.ContentCol className="autofit-col-toggle">
```

Comment:
Is switching between a button and a div while loading going to cause the user to lose focus if they had been interacting via the keyboard?

Reply:
Well in this case no because when you expand via keyboard the focus is always on the Cell and not on the button, so it's safe to do that.